traffic_ops/app/lib/API/Parameter.pm

```diff
@@ -317,7 +317,7 @@ sub update {
 
     my $name = $params->{name} || $find->name;
     my $configFile = $params->{configFile} || $find->config_file;
-    my $value = $params->{value} || $find->value;
+    my $value = exists($params->{value}) ?  $params->{value} : $find->value;
```

Comment:
I would like to allow to set the value to 0.

**Apparently in Perl the string "0" is equivalent to False....**
See http://www.perlmonks.org/?node=what%20is%20true%20and%20false%20in%20Perl%3F

I can replace the "exists" with "defined" if you prefer. I believe that logically "exists"is better, but it really does not matter for this case